akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -463,11 +536,13 @@ class ReplicatedDataSerializer(val system: ExtendedActorSystem)
   }
 
   def orsetFromProto(orset: rd.ORSet): ORSet[Any] = {
-    val elements: Iterator[Any] =
-      (orset.getStringElementsList.iterator.asScala ++
+    val elements: Iterator[Any] = {
+      orset.getStringElementsList.iterator.asScala ++
         orset.getIntElementsList.iterator.asScala ++
         orset.getLongElementsList.iterator.asScala ++
-        orset.getOtherElementsList.iterator.asScala.map(otherMessageFromProto))
+        orset.getOtherElementsList.iterator.asScala.map(otherMessageFromProto) ++
+        orset.getActorRefElementsList.iterator.asScala.map(resolveActorRef)
+    }
```

Comment:
regarding durable data (perhaps stored with an ancient version) it's no problem because otherElements contain full serialization info and can still be deserialized to right ActorRef.

Downgrading after using 2.5.15 with durable data is not supported, but that should be rather expected.

Reply:
Well, one could downgrade from 2.5.15 to 2.5.14, but not to 2.5.13.